frontend/src/helpers/validators/StaffFormValidations.ts

```diff
@@ -72,6 +72,27 @@ fieldValidations["businessInformation.lastName"] = [
   hasOnlyNamingCharacters("last name"),
 ];
 
+fieldValidations["businessInformation.businessName"] = [
+  isNotEmpty("Client name cannot be empty"),
+  isMaxSizeMsg("client name", 60),
+  isAscii("client name"),
+];
+
+fieldValidations["businessInformation.workSafeBcNumber"] = [
+  optional(isOnlyNumbers("WorkSafeBC number should contain only numbers")),
+  optional(isMaxSizeMsg("WorkSafeBC", 6))
+];
+
+fieldValidations["businessInformation.doingBusinessAs"] = [  
+  optional(isMaxSizeMsg("doing business as", 120)),
+  optional(isAscii("doing business as")),
```

Comment:
I'm not sure this field should allow any ASCII, like punctuations.

Reply:
Maybe it dues, because I would assume that I can have a doing business as called 'Acme Corp.' with the dot for example and it should in theory be acceptable